src/main/java/com/carmen/app/boundary/CarResource.java

```diff
@@ -1,20 +1,22 @@
 package com.carmen.app.boundary;
 
-import java.util.ArrayList;
+
+import java.util.LinkedHashMap;
```

Comment:
Could be a hashmap aswell instead of a LinkedHaspmap? That is an interesting question I think -> https://stackoverflow.com/questions/2889777/difference-between-hashmap-linkedhashmap-and-treemap